posts/apache-spark-3.0.md

```diff
@@ -0,0 +1,54 @@
+---
+authors:
+- Theo Lebrun
+tags:
+- Apache Spark
+- Big Data
+date: 2020-06-23T14:50:55.000Z
+title: "Apache Spark 3.0"
+image:
+---
+
+Databricks recently announced the release of [Apache Spark 3.0](https://databricks.com/blog/2020/06/18/introducing-apache-spark-3-0-now-available-in-databricks-runtime-7-0.html) with their Databricks Runtime 7.0. It's been a long road since the initial 1.0 release in 2014 which makes Spark one of the most active open-source big data projects with over 1000 contributors! This giant open-source community (one of the most active from the Apache Software Foundation) resolved over 3400 Jira tickets which is a pretty good way to celebrate its 10 years anniversary as an open-source project.
```

Comment:
```suggestion
Databricks recently announced the release of [Apache Spark 3.0](https://databricks.com/blog/2020/06/18/introducing-apache-spark-3-0-now-available-in-databricks-runtime-7-0.html) with their Databricks Runtime 7.0. It's been a long road since the initial 1.0 release in 2014 which makes Spark one of the most active open-source big data projects with over 1000 contributors! This giant open-source community (one of the most active from the Apache Software Foundation) resolved over 3400 Jira tickets which is a pretty good way to celebrate its 10 year anniversary as an open-source project.
```